libc/include/llvm-libc-types/jmp_buf.h

```diff
@@ -49,9 +53,22 @@ typedef struct {
 #endif
 #else
 #error "__jmp_buf not available for your target architecture."
+#endif
+  // TODO: implement sigjmp_buf related functions for other architectures
+  // Issue: https://github.com/llvm/llvm-project/issues/136358
+#if defined(__i386__) || defined(__x86_64__)
```

Comment:
Rather than conditionally enabling this based on architecture, we should do it based on the standards enabled in `hdrgen`. Specifically, this should only be enabled for `POSIX` and should never be included elsewhere regardless of the architecture.

Reply:
Ideally, we will need [`_POSIX_SOURCE`](https://www.gnu.org/software/libc/manual/html_node/Feature-Test-Macros.html) but I don't think this system is implemented in libc currently.